exchange/csvfeed.go

```diff
@@ -42,10 +44,11 @@ func (c CSVFeed) AssetsInfo(pair string) model.AssetInfo {
 	}
 }
 
-func NewCSVFeed(targetTimeframe string, feeds ...PairFeed) (*CSVFeed, error) {
+func NewCSVFeed(targetTimeframe string, candleType string, feeds ...PairFeed) (*CSVFeed, error) {
 	csvFeed := &CSVFeed{
 		Feeds:               make(map[string]PairFeed),
 		CandlePairTimeFrame: make(map[string][]model.Candle),
+		CandleType:          candleType,
```

Comment:
Maybe we can remove this parameter. I have the idea to keep it simple and avoid changes in functions signatures. A best approach is the inclusion of a bool parameter in `PairFeed` called `heikinashi`.

Pass a string with a candle ttype, makes me a little confused